etherpad-manage.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+min_stamp=$(date -d "-15 days" +%s)
+
+count=0
+
+for p in $(curl -s --data-urlencode "apikey=$ETHERPAD_API_KEY" "http://localhost:$PORT/api/1.2.1/listAllPads" | jq .data.padIDs[] | xargs echo); do 
+	last_updated=$(curl -s --data-urlencode "padID=$p" --data-urlencode "apikey=$ETHERPAD_API_KEY" "http://localhost:$PORT/api/1/getLastEdited" | jq '.data.lastEdited')
+	((last_stamp=last_updated/1000))
+	if [[ $last_stamp -lt $min_stamp ]]; then
+		echo $p is old and will be deleted
+		curl -s \
+			--data-urlencode "apikey=$ETHERPAD_API_KEY" \
+			--data-urlencode "padID=$p" \
+			"http://localhost:$PORT/api/1/deletePad"
+		((count++))
```

Comment:
If you do want the `set -e` above you'll want to change how this is calculated, maybe `count = $(( $count + 1 ))`

Reply:
okay, was trying to just copy over the file plus minimal changes to make it not break on weird characters. will add your bash ideas shortly.

Reply:
oh shoot, you can just ignore me! I always get confused with that kind of copy over vs new